fuel-tx/src/transaction.rs

```diff
@@ -102,6 +103,7 @@ pub enum Transaction {
     Upgrade(Upgrade),
     Upload(Upload),
     Blob(Blob),
+    Unknown,
```

Comment:
This kind of change should be done differently. From the protocol level, we don't have unknown transactions since it is impossible to do a state transition if the transaction is unknown.

A new variant should be handled by the code that may receive a new variant of the enum. Maybe you want to look into `fuel-core-client` since it is where you fetch transactions from the GraphQL.

Reply:
Thanks for the clarification.